README.md

```diff
@@ -16,9 +16,39 @@ limitations under the License.
 
 # <img src="logos/logo.svg" width="180">
 
-## Scalable Data Access Policy Management and Enforcement
+### A Tool for Complex and Scalable Data Access Policy Enforcement
 
-### Project Build
+## Overview
+
+From the client’s perspective, they submit a request to examine data and receive a reference for this data. 
+This reference can then be used to view the data after it has been retrieved and possibly redacted or filtered based on the context of the query and the permissions of the user making the request.
+
+Under the hood, the request involves sending a message that is processed sequentially by a set of the micro-services starting first with Palisade Service.
+With each service, the message is modified and then forwarded onto the next service with the end goal of producing resources ready to be viewed by the client.
+This initial sequence of steps is completed with the Filtered-Resource Service prepared to provide the processed data.
+In the subsequent request by the client to the Data Service, the Filtered-Resource is requested to then proceed with passing this data to the client.
+The following diagram shows in a high level the services and their relative relationships with each other. 
+Communications between the services and their related support servers utilise Kafka streaming. 
+
+For a detailed description of the services' workflow see [Appendix I - Palisade Flow Diagram](#appendix-i).  
+For a more technical description of each of the services, follow the links below.
+
+<!--- 
+See doc/services.drawio for the source of this diagram
+--->
+![Service Sequence diagram](doc/services.png)
```

Comment:
Data-Svc goes to AMS, not FRS
TOS goes from AMS to FRS as current, but AMS also goes straight to FRS (like a triangle shape between the three)
Audit Svc lines could be tidier

Reply:
Ok had another go at it.